templates/hooks/snapshot/sdk_delete_post_request.go.tpl

```diff
@@ -0,0 +1,17 @@
+if err == nil {
+	rp, _ := rm.setSnapshotOutput(r, resp.Snapshot)
+	// Setting resource synced condition to false will trigger a requeue of
+	// the resource.
+	ackcondition.SetSynced(
+		rp,
+		corev1.ConditionFalse,
+		&condMsgCurrentlyDeleting,
+		nil,
+	)
+	// Need to return a requeue error here, otherwise:
+	// - reconciler.deleteResource() marks the resource unmanaged
```

Comment:
What's wrong with marking it unmanaged? Do you want to keep reconciling it until it actually 404s? I think some controllers just let things disappear from k8s once they have confirmed it's in "deleting" status.